### DISCUSSION TITLE
Cancel a procedure

### DISCUSSION THREAD

**Discussion by mdewinne** [OP-TED/eForms-SDK#256] [Q&A] [answered]
We are unsure about the required approach to cancel a competition procedure before the submission deadline is reached and before the tenders have been opened.

**Scenario A - CA cancels the whole procedure (all lots)**

- Is it correct to assume that in this case, the CA needs to publish a CAN where BT-142 "Winner chosen" is set to "No winner was chosen and the competition is closed" for all lots? 
- And must BT-144 "Not Awarded Reason" also be completed in this case?
- If a CAN is used to fully cancel a procedure, is it still mandatory to provide all other information that is normally required for a CAN, e.g. notice value, the business groups for received submissions, tenders, contracts, etc. Supposedly, the business rules will exclude this, as logically the information won't be available?

**Scenario B - CA cancels only one of the lots (e.g. 3 lots total, buyer stops/cancels the first lot before the submission deadline is reached)**
We assume that the CA 
- Must publish a CAN, where for lot 1 BT-142 "Winner chosen" is set to "No winner was chosen and the competition is closed" and for all the other lots the same BT-142 is set to "The winner was not yet chosen, but the competition is still ongoing" ?
- Must provide a reason in BT-144 for lot 1?

What if the CA afterwards wants to publish a change notice for the remaining two lots (e.g. postpone the submission deadline). Can the first, by then cancelled lot, still be "present" in the "new" (change) contract notice? And if so, does it need to be flagged as "cancelled" in a way? Or does it need to be completely absent from the changed contract notice? 

Does the publication of the CAN to cancel one of the lots automatically requires the publication of a "new" (change) contract notice, to indicate that one of the lots will no longer be awarded?

**Comment by rousso** [comment, marked as answer]
Hi @mdewinne,

In GitHub discussions we are only able to respond to technical implementation issues.
This does not look like a technical question. **Please address business related questions to the OP helpdesk.**

Kind regards,
Yannis
